andela/titan-dev-ah-frontend
# #165969658 Fix Navigation Bar Picture

#### What does this PR do?
- Display correct image in navigation bar
#### Description of Task to be completed?
- Create the component for picture in navigation bar
- Create appropriate tests 
#### How should this be manually tested?
- Clone the repository and checkout the branch `ft-nav-bar-picture-165969658`
- Start the backend and front-end server 
- If you are logged in you should see the profile picture, username, or first name and last name in the navigation bar
- If you are not logged in, you should see the default profile picture only. 

#### What are the relevant pivotal tracker stories?
# 165969658
